src/components/base/JCard/JCard.stories.js

```diff
@@ -3,7 +3,10 @@
 import React from 'react'
 import { storiesOf } from '@storybook/react'
 
-import { JCard, JText } from '../../../src/components/base'
+import {
```

Comment:
Import component *Case4*

I guess, we need some rule of imports components in stories